Project Now Has Like Settings

NielsSensei opened the pull request:
Kunt ge selecteren met een comboboxke voor de versch types.
Als je niks selecteert is de standaard setting: Alle Type Likes.

Je kan dit kiezen in add & change project.

Ik heb ook _AddReactionNumberPartial ondersteboven gesmeten dat alle soorten likes met icoontjes daar nu zitten en ook Settings die niet perse behoren tot Project worden afgehandeld hier. Alles is juist aangeroepen en al.

NielsSensei committed:
Going to check out how it used to look

Changed Prop Likevisibility to enum.

In Add & Change Project een ddl op basis van deze enum om deze te setten.

In projectView wordt de likepartial oek aangepast a.d.h.v deze setting.

NielsSensei committed:
Pre PR recheck.

Also removed TODO's, we got youtrack & communication for that

NielsSensei committed:
pre merge

NielsSensei committed:
Merge branch 'master' into feature/admin/like-visibility

NielsSensei committed:
part 4 done

XanderVeldeman reviewed (changes requested):
*Kijk mama, nu ben ik ook een boeman!*
De routing voor het aanpassen van de Likes klopt niet
![image](https://user-images.githubusercontent.com/25788097/58047418-74443680-7b48-11e9-94e7-57e202fde6e3.png)

![image](https://user-images.githubusercontent.com/25788097/58047447-82925280-7b48-11e9-9a40-26980fdb0170.png)

Kan je ook zorgen voor een knop om een project aan te passen?

NielsSensei committed:
Fixes

Error Routing verbeterd
Visible (project) gefixt
Juiste redirect als project invis is nu.

NielsSensei committed:
More hotfixes

NielsSensei committed:
this also needed to be added

XanderVeldeman committed:
fix for project list check
